src/Admin/Preview.php

```diff
@@ -501,6 +501,14 @@ public function post_to_preview_instance( $post_ID, $post ) {
 
 		$original_post = get_post( $post->post_parent );
 
+		$this_is_a_publish_not_a_preview = 
```

Comment:
@jasonbahl can you think of any problems with this? From my testing it works great. If we don't do this, we'll send a preview + a save post action at the same time (both to the preview instance).